string_assignment.js

```diff
@@ -0,0 +1,66 @@
+//STRING EXERCISES
```

Comment:
Overall good demonstration of how to use string properties and methods to solve problems. Keep practicing and keep trying to answer bonus questions.